src/modules/site-v2/templates/tools/heritability_calculator/result.html

```diff
@@ -196,17 +196,17 @@
     }
 
     function getRandomColor() {
-        var letters = '123456789ABCDEF'.split('');
-        var color = '#';
-        for (var i = 0; i < 6; i++ ) {
+        const letters = '123456789ABCDEF'.split('');
```

Comment:
Do we need to worry about contrast against background/other elements on the page? If so, we might need to restrict the range of the output color based on some constraints.

Reply:
Good question -- that might be a question for Natalie.